AdminUi/src/AdminUi/Controllers/IdentitiesController.cs

```diff
@@ -103,10 +100,10 @@ public async Task<IActionResult> GetIdentityByAddress([FromRoute] string address
     [HttpPut("{identityAddress}")]
     [ProducesResponseType(StatusCodes.Status204NoContent)]
     [ProducesError(StatusCodes.Status404NotFound)]
-    public async Task<IActionResult> PutIdentityTierByAddress([FromRoute] string identityAddress, [FromBody] UpdateIdentityTierRequest request, CancellationToken cancellationToken)
+    public async Task<IActionResult> PutIdentityByAddress([FromRoute] string identityAddress, [FromBody] UpdateIdentityTierRequest request, CancellationToken cancellationToken)
     {
         var command = new UpdateIdentityCommand() { Address = identityAddress, TierId = request.TierId };
-        await _mediator.Send<Backbone.Modules.Devices.Domain.Entities.Identity>(command, cancellationToken);
+        await _mediator.Send<Identity>(command, cancellationToken);
```

Comment:
Just noticed: Why the hell do you return an Identity from the command??? We NEVER EVER return domain objects from the application layer! That's one of the most important rules.
And you don't even use the return value here anyway.